compiler/damlc/stable-packages/src/GenerateStablePackage.hs

```diff
@@ -581,6 +583,54 @@ daInternalPromotedText = package version1_6 $ NM.singleton Module
       [ DefDataType Nothing ptextTyCon (IsSerializable False) [(mkTypeVar "t", KStar)] $ DataVariant []
       ]
 
+daInternalError :: Package
+daInternalError = Package
+    { packageLfVersion = versionDev
```

Comment:
can we use `featureMinVersion featureExceptions` instead?

Reply:
Ah, yeah :)